rpc/wrtc_signaling_test.go

```diff
@@ -186,6 +187,16 @@ func TestWebRTCAnswererImmediateStop(t *testing.T) {
 
 	// Running both asynchronously means Stop will potentially happen before Start,
 	// but this setup still ensures that the two methods do not race each other.
-	go answerer.Start()
-	go answerer.Stop()
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	go func() {
+		defer wg.Done()
+		answerer.Start()
+	}()
+	go func() {
+		defer wg.Done()
+		answerer.Stop()
+	}()
+	wg.Wait()
```

Comment:
@maximpertsov, I felt the original timeout of 30 seconds was well intentioned, but then we lose the stack traces of what's happening if the test were to fail. And without those stacks we'd never understand why* `Wait` didn't return. I don't think there's any real value in trying to protect CI from spinning for 10 minutes before failing, so I urged Pranav to strip that bit out.

Reply:
fair point, that works for me